tests/tests-cart.php

```diff
@@ -721,4 +724,12 @@ public function test_negative_fees_cart_tax() {
 
 		edd_update_option( 'enable_taxes', false );
 	}
+
+	public function test_cart_is_empty() {
+		edd_add_to_cart( self::$download->ID, array( 'price_id' => 0 ) );
+		$this->assertFalse( edd_is_cart_empty() );
+
+		edd_empty_cart();
+		$this->assertTrue( edd_is_cart_empty() );
```

Comment:
These two should get broken off into their own 'test'. This test can be renamed to `test_cart_is_empty_false`, and then add these two lines to a method called `test_cart_is_empty_true`.

Isolating assertions to as few as possible in a 'test' make it easier to debug when things go wrong. When a single assertion fails, the entire test is marked as failed, even if something else in it passes.